.murdock

```diff
@@ -7,65 +7,31 @@
 #: ${TEST_BOARDS_LLVM_COMPILE:="iotlab-m3 native nrf52dk mulle nucleo-f401re samr21-xpro slstk3402a"}
 : ${TEST_BOARDS_LLVM_COMPILE:=""}
 
-: ${TEST_KCONFIG_samr21_xpro:="examples/hello-world tests/periph_*
-tests/test_tools tests/congure_* tests/xtimer_* tests/ztimer_*
-tests/driver_ad7746 tests/driver_adcxx1c tests/driver_ads101x tests/driver_adt101x
-tests/driver_adt7310 tests/driver_adxl345 tests/driver_aip31068 tests/driver_apa102
-tests/driver_apds99xx tests/driver_apds99xx_full tests/driver_at tests/driver_at24cxxx
-tests/driver_at24mac tests/driver_at25xxx tests/driver_at30tse75x tests/driver_ata8520e
-tests/driver_b* tests/driver_ccs811 tests/driver_ccs811_full tests/driver_dcf77
-tests/driver_dfplayer tests/driver_dht tests/driver_ds18 tests/driver_ds75lx
-tests/driver_ds1307 tests/driver_ds3231 tests/driver_ds3234 tests/driver_dsp0401
-tests/driver_dynamixel tests/driver_edbg_eui tests/driver_f* tests/driver_g*
-tests/driver_h* tests/driver_i* tests/driver_j* tests/driver_l*
-tests/driver_mag3110 tests/driver_mhz19 tests/driver_mma7660
-tests/driver_motor_driver tests/driver_mpl3115a2 tests/driver_mpu9x50
-tests/driver_mq3 tests/driver_my9221 tests/driver_nvram_spi tests/mtd_flashpage
-tests/mtd_mapper tests/driver_o* tests/driver_p* tests/driver_q*
-tests/driver_r* tests/driver_s* tests/driver_t* tests/driver_u*
-tests/driver_v*"}
-: ${TEST_KCONFIG_native:="examples/hello-world
+: ${TEST_KCONFIG_BOARDS_AVAILABLE:="
+native
+samr21-xpro
+"}
+
+: ${TEST_KCONFIG_ENFORCE_APP_GROUPS:="
 tests/cb_mux*
 tests/congure_*
-tests/driver_ws281x
```

Comment:
This one seems to have been lost

Reply:
It is automatically included as it has a app.config.test file

Reply:
ahh sorry for the noise!